### DISCUSSION TITLE
web可登录，ssh无法连接

### DISCUSSION THREAD

**Discussion by zlopen** [immortalwrt/immortalwrt#985] [Q&A]
### Describe the bug

luci web界面可以登录上
ssh时luci上能看到日志
Mon Jun 19 23:20:19 2023 authpriv.info dropbear[14214]: Child connection from 192.168.100.169:10100
Mon Jun 19 23:20:19 2023 authpriv.warn dropbear[14214]: User 'root' has invalid shell, rejected
Mon Jun 19 23:20:25 2023 authpriv.info dropbear[14214]: Exit before auth from <192.168.100.169:10100>: (user 'root', 1 fails): Error reading: Connection reset by peer
如果通过luci-app-command能看到一些信息
# 'cat' '/etc/passwd'
root:x:0:0:root:/root:/bin/bash
...
# 'find' / '-name' 'bash'
/bin/bash
...
# 'ls' '/dev/tty*'
ls: /dev/tty*: No such file or directory

ssh登录不上，情况如下
debug2: we did not send a packet, disable method
debug3: authmethod_lookup password
debug3: remaining preferred: ,password
debug3: authmethod_is_enabled password
debug1: Next authentication method: password
debug3: failed to open file:C:/dev/tty error:3
debug1: read_passphrase: can't open /dev/tty: No such file or directory
root@192.168.100.1's password:
debug3: send packet: type 50
debug2: we sent a password packet, wait for reply
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password
Permission denied, please try again.
debug3: failed to open file:C:/dev/tty error:3
debug1: read_passphrase: can't open /dev/tty: No such file or directory
root@192.168.100.1's password:

### ImmortalWrt version

r0-ffd175e

### ImmortalWrt target/subtarget

ipq807x/generic

### Device

Xiaomi AX3600 (stock layout)

### Image kind

Official downloaded image

### Steps to reproduce

分支地址
https://github.com/immortalwrt/immortalwrt/tree/openwrt-23.05
编译刷机

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
你的编译环境或固件有问题，bash 不能正常工作。
可以自行修改 passwd 文件把 shell 改成 `/bin/ash`，但建议重刷固件。

**Comment by laaArmy** [comment]
你好
问题解决了没？
